doc/releases/changelog-0.40.0.md

```diff
@@ -6,28 +6,80 @@
 
 <h4>Efficient state preparation methods 🦾</h4>
 
-* Added new ``MPSPrep`` template to prepare quantum states in tensor simulators.
+* State preparation tailored for matrix product states (MPS) is now supported with 
+  :class:`qml.MPSPrep <pennylane.MPSPrep>` on the `lightning.tensor` device.
   [(#6431)](https://github.com/PennyLaneAI/pennylane/pull/6431)
 
-* Users can prepare a linear combination of basis states using `qml.Superposition`.
+  Given a list of :math:`n` tensors that represents an MPS, :math:`[A^{(0)}, ..., A^{(n-1)}]`, 
+  :class:`qml.MPSPrep <pennylane.MPSPrep>` lets you directly inject the MPS into a QNode as the initial 
+  state of the circuit without any need for pre-processing. The first and last tensors in the list 
+  must be rank-2, while all intermediate tensors should be rank-3.
+
+  ```python
+  import pennylane as qml
+  import numpy as np
+
+  mps = [
+      np.array([[0.0, 0.107], [0.994, 0.0]]),
+      np.array(
+          [
+              [[0.0, 0.0, 0.0, -0.0], [1.0, 0.0, 0.0, -0.0]],
+              [[0.0, 1.0, 0.0, -0.0], [0.0, 0.0, 0.0, -0.0]],
+          ]
+      ),
+      np.array(
+          [
+              [[-1.0, 0.0], [0.0, 0.0]],
+              [[0.0, 0.0], [0.0, 1.0]],
+              [[0.0, -1.0], [0.0, 0.0]],
+              [[0.0, 0.0], [1.0, 0.0]],
+          ]
+      ),
+      np.array([[-1.0, -0.0], [-0.0, -1.0]]),
+  ]
+
+  dev = qml.device("lightning.tensor", wires = 3)
```

Comment:
@KetpuntoG you're saying this should be `wires=4`?

Reply:
I just see this too https://xanaduhq.slack.com/archives/C06T4AFPR5E/p1736361417729549

Reply:
```suggestion
  dev = qml.device("lightning.tensor", wires = [0, 1, 2, 3])
```